package.json

```diff
@@ -4,8 +4,10 @@
   "description": "really simple connector for redux",
   "main": "index.js",
   "scripts": {
-    "start": "budo example/index.js",
-    "test": "browserify test.js | testron | tap-spec"
+    "start": "budo examples/simple/index.js",
+    "test": "browserify test.js | testron | tap-spec",
+    "start:simple": "budo examples/simple/index.js",
+    "start:deep-components": "budo examples/deep-components/index.js"
```

Comment:
i would call deep-component -> connect, cause then i might just have three examples, display, dispatch, and connect